nav/nav_stack/package.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>nav_stack</name>
+  <version>0.0.0</version>
+  <description>The nav_stack package</description>
```

Comment:
Description should use the full term `Navigation Stack`. eg. "Configures and runs the Navigation Stack through `move_base`".